envi/archs/h8/emu.py

```diff
@@ -0,0 +1,1314 @@
+"""
+The H8 Emulator module.
+"""
+
+import sys
+import struct
+
+import emu
+import envi
+import envi.bits as e_bits
+from envi.archs.h8 import H8Module
+from envi.archs.h8.regs import *
+from disasm import H8RegDirOper
+
+# calling conventions
+class H8Call(envi.CallingConvention):
+    """
+    Implement calling conventions for your arch.
```

Comment:
Seems like this would be a great spot to acutally doc the calling convention ;) ( but not a show stopper )